DOL.WHD.Section14c.Test/Business/BaseAPIControllerTest.cs

```diff
@@ -46,5 +52,43 @@ public void BaseApiController_ExpectationFailed_Test()
         {
             ExpectationFailed("Expectation Failed");
         }
+
+        [TestMethod()]
+        public void MyHttpClientTest()
+        {
+            var httpClientInstance = MyHttpClient;
+            Assert.IsNotNull(httpClientInstance);
+        }
+
+        [TestMethod()]
+        public void DownloadTest()
+        {
+            // Arrange
+            var testFileContents = "test";
+            var data = Encoding.ASCII.GetBytes(testFileContents);
+            var memoryStream = new MemoryStream(data);
+            var fileName = "test.pdf";
+
+            var request = new Mock<HttpRequestMessage>();
+            HttpResponseMessage response = request.Object.CreateResponse(HttpStatusCode.OK);
+            var resp = Download(data, response, fileName);
+            Assert.AreEqual(resp.StatusCode, HttpStatusCode.OK);
```

Comment:
Please also assert the content and the content-type, content-length, and content-disposition headers.

Reply:
This one is done in #419